src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -48,13 +48,8 @@ def __init__(self, streams, stage, area, overlap, settings_obs=None, log_path=No
         self._streams = streams
 
         # Get total area as a tuple of width, height from ltrb area points
-        if len(area) != 4:
-            raise ValueError("4 points required (left, top, right, bottom) for total acquisition area.")
-        if not (area[1] > area[3]):
-            raise ValueError("Top point {0} should be larger than bottom point {1}.".format(area[3], area[1]))
-        if not (area[2] > area[0]):
-            raise ValueError("Right point {0} should be larger than left point {1}.".format(area[2], area[0]))
-        height = area[1] - area[3]
+        area = util.normalize_rect(area)
+        height = area[3] - area[1]
```

Comment:
I replaced these checks with `util.normalize_rect()` since it already checks for the `len(area)` and reorder the parameters.